tests/ui/test_devhub.py

```diff
@@ -57,3 +61,28 @@ def test_devhub_register(base_url, selenium):
     assert not devhub.logged_in
     devhub.header.register()
     assert 'signup' in selenium.current_url
+
+
+@pytest.mark.fxa_login
+@pytest.mark.desktop_only
+@pytest.mark.nondestructive
+@pytest.mark.withoutresponses
+def test_devhub_addon_upload_approve_install(
+        base_url, selenium, devhub_upload, firefox, firefox_notifications):
+    """Test uploading an addon via devhub."""
+    'ui-test-addon-2' in devhub_upload.addons[-1].name
+    # We have to wait for approval
+    time.sleep(15)
+    selenium.get('{}/addon/{}'.format(base_url, 'ui-test_devhub_ext/'))
```

Comment:
Unsure if maybe https://docs.python.org/2/library/urlparse.html#urlparse.urljoin leads to a bit nicer code here… but given that we're joining two paths onto the base url it might not…

Reply:
I could just add the second path to the string so its just one set of ```{}``` instead of 2